plugin/trino-ranger/README.md

```diff
@@ -0,0 +1,95 @@
+# Trino-Ranger plugin
+
+This plugin is designed to be build and run inside Trino.
+
+It works with vanilla Apache Ranger, version 2.2.1 and up. You dont need to customize Ranger at all.
+
+Here are the setup steps.
+
+
+### Full Enterprise ready Apache Ranger setup
+
+ * Install and startup Ranger
+   * Quick and dirty setup for Ranger
+     * git clone Ranger. I recommend using official tagged released. Here is an example:
+       * git clone --recursive --branch release-ranger-2.2.0  https://github.com/apache/ranger
+     * Build Ranger. Here is the commands I use
+       * mvn -e -X clean package -pl '!plugin-kylin,!ranger-kylin-plugin-shim'  -DskipTests
+     * You are looking for all of the build targets under "target/ranger-*.tar.gz"
+     * Copy the main Ranger file to your binary install directory and expand it. Lets call this directory $RANGER_HOME
+       * The main service to install is the ranger-$RANGER_VERSION-admin.tar.gz
+     * Ranger has 2 configuration files that generate the full ranger install
+       * $RANGER_HOME/install.properties
+         * You need to generate this config. There is already a sample install.properties extracted from the admin.tar.gz file
+           * Most of the defaults work fine. The configs you want to pay attention to are
+             * PYTHON_COMMAND_INVOKER, DB_FLAVOR, SQL_CONNECTOR_JAR, db_root_user, db_root_password, db_host, db_name, db_user, db_password, rangerAdmin_password, rangerTagsync_password, rangerUsersync_password, keyadmin_password
+       * $RANGER_HOME/usersync/install.properties
+         * In most enterprise environments, ActiveDirectory/LDAP are used to manage users and their roles.
```

Comment:
We should likely have a separate section for LDAP that we reference here. That would make this section shorter and also leave it up to the reader if they want to just play around they can do as they please. I think the recommendation is sound but not everyone is ready to immediately jump into an LDAP setup.

Reply:
> We should likely have a separate section for LDAP

Agreed. This should be 2 separate setups. LDAP or stand-alone.